Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM node:8.4.0-alpine
 
+RUN apk add --no-cache python=2.7.13-r1 git-perl bash make gcc g++
```

Comment:
@markysoft can you explain please?

Reply:
Brunch relies on node-sass, which requires python.
node-sass bundles a number of pre-compiled binaries for popular distributions, but not the flavour of linux we use.

I've added lines to the Docker file to install the pre-requisites as decribed in the issue https://github.com/sass/node-sass/issues/1589

The issue has been fixed in master on node-sass, but it has not been released since May